task5/src/main/java/com/clouway/adapter/http/BankOperationHandler.java

```diff
@@ -35,10 +38,17 @@ public BankOperationHandler(FundsRepository fundsRepository, Validator validator
   protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
     Cookie[] cookies = request.getCookies();
     Cookie cookie = cookieFinder.find(cookies);
+    if (cookie == null) {
+      response.sendRedirect("/login?errorMsg=Session expired! Please log in again!");
```

Comment:
I found a bug  when i've already closed the previous PR. This modification solve my problem.